### PR TITLE
Init_model

### PR DESCRIPTION
**Pull request by nguyenvanson332000**
## Checklist
- [ ] This is bug/feature (delete incorrect one)
- [ ] Title including ticket number and title

## What this PR do?
Explain clearly and briefly what this PR does. Make a meaningful sentence instead of "this PR solve ticket #xxx"

## Context/Background
More information on the issue that you want to provide, related issues and PRs you have to write information to ask orally or in informal place too.

## Screenshots / Demo & Instruction for testing (if applicable)
- Screenshots
- link to demo
- instruction on how to test this

## References
- Blog posts / web pages relevant to this issue (which you get your solution / idea from and it may help others to understand the issue quickly)
- Do we need update our knowledge base / project wiki?

### CODE REVIEW

File: app/models/question.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class Question < ApplicationRecord
+  has_many :answers, dependent: :destroy
+  has_many :exams, dependent: :destroy
```
Comment (rikai-hai) [inline app/models/question.rb]: ??? question xóa liên quan gì xóa exams

File: db/migrate/20220407042407_create_answers.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+class CreateAnswers < ActiveRecord::Migration[6.0]
+  def change
+    create_table :answers do |t|
+      t.references :question, foreign_key: true
+      t.string :content
+      t.boolean :is_correct_answer
```
Comment (rikai-hai) [inline db/migrate/20220407042407_create_answers.rb]: thêm default option

File: db/migrate/20220407042625_create_exams.rb [hunk 3]
```diff
@@ -0,0 +1,13 @@
+class CreateExams < ActiveRecord::Migration[6.0]
+  def change
+    create_table :exams do |t|
+      t.references :question, foreign_key: true
```
Comment (rikai-hai) [inline db/migrate/20220407042625_create_exams.rb]: 1 exams nhiều question chứ hè ?

File: db/migrate/20220407042625_create_exams.rb [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+      t.string :exam_name
```
Comment (rikai-hai) [inline db/migrate/20220407042625_create_exams.rb]: name thôi

File: db/migrate/20220407042625_create_exams.rb [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+      t.string :time_limit
+      t.float :pass_score
```
Comment (rikai-hai) [inline db/migrate/20220407042625_create_exams.rb]: integer

File: db/migrate/20220407042625_create_exams.rb [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+      t.integer :number_question
```
Comment (rikai-hai) [inline db/migrate/20220407042625_create_exams.rb]: cái này làm gì ?

File: db/migrate/20220407042919_create_user_exams.rb [hunk 7]
```diff
@@ -0,0 +1,14 @@
+class CreateUserExams < ActiveRecord::Migration[6.0]
+  def change
+    create_table :user_exams do |t|
+      t.references :exam, foreign_key: true
+      t.references :user, foreign_key: true
+      t.float :total_score
+      t.time :completed_time
+      t.boolean :is_passed
```
Comment (rikai-hai) [inline db/migrate/20220407042919_create_user_exams.rb]: default option

File: test/fixtures/answers.yml [hunk 8]
```diff
@@ -0,0 +1,11 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```
Comment (rikai-hai) [inline test/fixtures/answers.yml]: bỏ hết cái file kiểu này

File: test/models/user_exam_test.rb [hunk 9]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```
Comment (rikai-hai) [inline test/models/user_exam_test.rb]: bỏ hết các file liên quan đến test

File: db/migrate/20220407042625_create_exams.rb (hunk 3 above)
Comment (nguyenvanson332000) [reply db/migrate/20220407042625_create_exams.rb]: Dạ để em bổ sung ạ

File: db/migrate/20220407042625_create_exams.rb (hunk 4 above)
Comment (nguyenvanson332000) [reply db/migrate/20220407042625_create_exams.rb]: Dạ

File: test/fixtures/answers.yml (hunk 8 above)
Comment (nguyenvanson332000) [reply test/fixtures/answers.yml]: Dạ

File: test/models/user_exam_test.rb (hunk 9 above)
Comment (nguyenvanson332000) [reply test/models/user_exam_test.rb]: Dạ

File: db/migrate/20220407042625_create_exams.rb (hunk 5 above)
Comment (nguyenvanson332000) [reply db/migrate/20220407042625_create_exams.rb]: Dạ

File: db/migrate/20220407042625_create_exams.rb (hunk 6 above)
Comment (nguyenvanson332000) [reply db/migrate/20220407042625_create_exams.rb]: à dạ cái đấy dư, để em xóa

**Pull request merged**